.github/workflows/server-ci.yml

```
@@ -40,10 +40,10 @@ jobs:
         uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd  # v6.0.2
         with:
           persist-credentials: false
-      - name: Calculate version
+      - name: Calculate build image tag
         id: calculate
         working-directory: server/
-        run: echo GO_VERSION=$(cat .go-version) >> "${GITHUB_OUTPUT}"
+        run: echo BUILD_SERVER_TAG=$(./build/buildenv-tag.sh) >> "${GITHUB_OUTPUT}"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:23: Quote this to prevent word splitting

(shellcheck)